Sofa/framework/Core/src/sofa/core/objectmodel/BaseContext.h

```diff
@@ -414,4 +404,23 @@ class SOFA_CORE_API BaseContext : public virtual Base
     ComponentNameHelper m_nameHelper;
 };
 
+template<class T, class Container>
+class BaseContext::GetObjectsCallBackT : public BaseContext::GetObjectsCallBack
```

Comment:
could be the good timing for a really short doxygen description of this class

Reply:
@alxbilger ?

Reply:
done. Thanks for the reminder